spec/Export/Field/CustomFieldsSpec.php

```diff
@@ -59,7 +59,7 @@ function let(
         SalesChannelContext $channelContext,
         EntityRepositoryInterface $customFieldRepository,
         EntityRepositoryInterface $languageRepository,
-        ExcludedFields $excludedFields,
+        ExportSettings $excludedFields,
```

Comment:
can we change this to `$exportSettings` since the class name has been changed too?